examples/chat/README.md

```diff
@@ -1,57 +1,35 @@
-# Jazz Todo List Example
+# Jazz Chat Example
 
-Live version: https://example-todo.jazz.tools
+Live version: https://example-chat.jazz.tools
 
 ## Installing & running the example locally
 
-Start by checking out just the example app to a folder:
+(this requires `pnpm` to be installed, see [https://pnpm.io/installation](https://pnpm.io/installation))
 
+Start by checking out `jazz`
 ```bash
-npx degit gardencmp/jazz/examples/todo jazz-example-todo
-cd jazz-example-todo
+git clone https://github.com/gardencmp/jazz.git
+cd jazz/examples/chat
```

Comment:
this can be simplified by running `pnpm dev-chat` in the root
https://github.com/gardencmp/jazz/pull/152/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R18

we also have `dev-chat-passphrase` `dev-todo` atm, feel free to rename

Reply:
ah sorry, i understand now what you want to achieve